lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -539,16 +560,18 @@ void OpenCLFunction::execute() {
       size_t global;
       if (I.isDataParallel()) {
         global = I.getOperand(0).first->getType()->size();
-        if (global % 16 == 0) {
-          // Start less kernels and let each kernel do more work using vector
-          // instructions.
-          global /= 16;
-          kernelName += "16";
-        } else if (global % 8 == 0) {
-          // Start less kernels and let each kernel do more work using vector
-          // instructions.
-          global /= 8;
-          kernelName += "8";
+        if (!isQuantized) {
```

Comment:
Can you add a comment that this a temporary workaround until the corresponding kernels are implemented for the quantized operations? Or may be you do not plant to implement them?

Reply:
Comments added. I probably would submit a following pr to add the corresponding operations